### ISSUE TITLE
[Planning] Ensure Docs-Community SIG requirements are identified

### ISSUE THREAD

**Issue by forhalle** [o3de/o3de#6792]
Work with the Docs-Community SIG to define requirements, and determine if any of the requirements will block the release if not completed prior to the release. These requirements could include, for example, release notes, tutorials, etc.

Discord: [#sig-docs-community](https://discord.gg/FxmvF5x8)
SIG Docs-Community meetings: [O3DE Calendar](https://lists.o3de.org/g/o3de-calendar/calendar)
SIG Docs-Community agendas: [GHI](https://github.com/o3de/community/issues)

**Comment by forhalle** [comment]
I asked @sptramer via. #sig-docs-release how I can get on the SIG's agenda.

**Comment by sptramer** [comment]
Releases are never blocked on docs. This includes release notes. Nothing prevents the release from going public and being published as binaries / on GitHub.

Also, release notes are the responsibility of sig-release, not sig-docs-community.

**Comment by sptramer** [comment]
Reopening since I have no idea what the end result of this issue should be.

**Comment by forhalle** [comment]
I requested identification of any community requirements in #sig-docs-community in [this](https://discord.com/channels/805939474655346758/816043656377008179/930234093050949692) and [this ](https://discord.com/channels/805939474655346758/816043656377008179/930514855302168596)post, but did not recieve a response. I'm therefore closing this issue as per my comment in [this ](https://discord.com/channels/805939474655346758/816043656377008179/931671652905926719)post.

**Comment by forhalle** [comment]
> 

I have removed "release notes" from the description of this ticket, and have created a new ticket (#6926) specifically for release notes.

**Comment by forhalle** [comment]
Note: At today's Sig Release meeting, @Ulrick28 confirmed there will be no tutorials, blog posts, or other community requirements.
